packages/@sfcc-bff/productapi/src/api/models/Product.js

```diff
@@ -213,14 +212,16 @@ class Product {
             userSelectedColor !== 'undefined' && userSelectedColor !== 'null'
                 ? userSelectedColor
                 : null;
+
         this.images = getImages(apiProduct.imageGroups, selectedColor);
 
         Object.assign(this, apiProduct);
         this.longDescription = apiProduct.longDescription;
         this.shortDescription = apiProduct.shortDescription;
 
         // Set a default image
-        this.image = get(apiProduct, 'this.images[0].link');
```

Comment:
This is a bug currently in the app, which was exposed when I upgraded our version of graphql. Essentially before the `image` property was literally a string that was `"[Object object]"`. This now properly gets the first image link.